app/controllers/api/arbitration_profiles_controller.rb

```diff
@@ -3,8 +3,8 @@ class ArbitrationProfilesController < BaseController
     def create_resource(_type, _id, data)
       validate_profile_data(data)
       attributes = build_arbitration_attributes(data)
-      arbitration_profile = collection_class(:arbitration_profiles).create(attributes)
-      validate_profile(arbitration_profile)
+      arbitration_profile = ArbitrationProfile.create(attributes)
+      raise BadRequestError, arbitration_profile.errors.full_messages.join(', ') unless arbitration_profile.valid?
```

Comment:
Could you prepend back "Failed to add new arbitration profile - " to the error message ?   Thanks